### ISSUE TITLE
GIVstream

### ISSUE THREAD

**Issue by divine-comedian** [Giveth/giveth-planning#310]

**Comment by laurenluz** [comment]
ok, this is ready for QA @divine-comedian @WhyldWanderer @hannahjojo 

FYI, I'm going to add an info graphic showing the "expanding GIViverse"... basically concentric circles with lines radially out. Going to get Rodri to help w that

**Comment by laurenluz** [comment]
https://hackmd.io/AFt8g5A6T7C9XVI_06jc4w?both

**Comment by divine-comedian** [comment]
I gave it a quick review and left some comments, the text is mostly good, it will be useful to get some uninitiated eyes to read this document as all of us already know how this system works.

**Comment by laurenluz** [comment]
@WhyldWanderer @ForestSoleil @divine-comedian please review:
https://docs.google.com/document/d/1ArudRhCo7MWY5g8JpawKDsR22nH2Nqkfhl8M_0RvNcU/edit?usp=sharing

### References
FYI - here's the HackMD if you wanna see the pics/formatting
https://hackmd.io/AFt8g5A6T7C9XVI_06jc4w?both

Also... If anyone wants to ratify my dates and make sure I did math good: https://docs.google.com/spreadsheets/d/1fgZl3ZeGH4CbixKDwfDmy7c3GKlSbepuchw22QrzNz8/edit#gid=0

**Comment by divine-comedian** [comment]
reviewed! In general I think we should be clear that the GIVstream is a function of time primarily but can be augmented by increasing your contributions - a universe expands infinitely with time - let's just be sure that our narratives don't .. cross streams  (hahahahaha)

**Comment by laurenluz** [comment]
@divine-comedian it's tricky to pick the best framing for sure... but this giviverse thing is all over the place already. GIVdrop, several pages of the GIVeconomy, multiple pieces of documentation, the launch article and the video and I don't have the bandwidth or motivation to change it now.

Leaving it as-is for now. If this is something we want to change after launch we can do so then... especially if we get feedback from users that are confused.

I think the GIViverse concept is fine... it's like everyone has a GIVstream and they can augment it with increased contributions... but the GIViverse expands regardless of each individuals contributions... all tokens are becoming liquid at the same rate... and if they;re not flowing to users, they're flowing to different pools and becoming liquid there (i.e. the undistributed rewards in the GIVfarm).

Long story short, I think it's not terrible framing and that it's too late in the game to make that change across the GIViverse and get everything else ready for launch 👀

**Comment by ForestSoleil** [comment]
Reviewed - looks good and appreciate the hackmd graphics

**Comment by WhyldWanderer** [comment]
Reviewed! i left some suggestions in the google doc

**Comment by divine-comedian** [comment]
LIVE! 

https://docs.giveth.io/giveconomy/givstream
(just missing links)
